src/js/components/Notification/index.d.ts

```diff
@@ -1,12 +1,27 @@
 import * as React from 'react';
 
 export type StatusType = 'critical' | 'warning' | 'normal' | 'unknown';
+export type PositionType =
```

Comment:
Can we just import `LayerPositionType` and re-use it here.

Reply:
yes good call